web/portal/brand/src/entities/Company/Form.tsx

```diff
@@ -104,7 +109,10 @@ const Form = (props: EntityFormProps): JSX.Element | null => {
     edit &&
       !isResidential && {
         legend: _('Platform data'),
-        fields: ['outgoingDdi', 'outgoingDdiRule'],
+        fields: [
+          !isWholesale && !isRetail && 'outgoingDdi',
+          !isWholesale && !isRetail && 'outgoingDdiRule',
```

Comment:
These could also be represented in the positive form
```
         (isVpbx || isResidential) && 'outgoingDdi',
         (isVpbx || isResidential) && 'outgoingDdiRule',
```

But this section "Platform data" is not displayed for Residentials, so these are just:
```
         isVpbx && 'outgoingDdi',
         isVpbx && 'outgoingDdiRule',
```